extensions/security-jpa/deployment/src/test/resources/multitenant-persistence-unit/application.properties

```diff
@@ -0,0 +1,19 @@
+quarkus.datasource.db-kind=h2
+quarkus.datasource.username=sa
+quarkus.datasource.password=sa
+quarkus.datasource.jdbc.url=jdbc:h2:mem:default
+
+quarkus.datasource.one.db-kind=h2
+quarkus.datasource.one.username=sa
+quarkus.datasource.one.password=sa
+quarkus.datasource.one.jdbc.url=jdbc:h2:mem:one
+
+quarkus.datasource.two.db-kind=h2
+quarkus.datasource.two.username=sa
+quarkus.datasource.two.password=sa
+quarkus.datasource.two.jdbc.url=jdbc:h2:mem:one
```

Comment:
Shouldn't this be:

```suggestion
quarkus.datasource.two.jdbc.url=jdbc:h2:mem:two
```

Reply:
That would mean I have to create user table manually or setup another PU. I've renamed it to `shared`. FYI verification that multitenancy works is done by dynamically setting `unknown` tenant which leads to error. I don't need different database for this.